sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/implementation/jackson/SerializerUtils.java

```diff
@@ -30,6 +30,10 @@ public class SerializerUtils {
         OBJECT_MAPPER.registerModule(new JavaTimeModule());
     }
 
+    private SerializerUtils(){
+        throw new IllegalStateException("Utility class");
```

Comment:
only private ctor is enough, why add the exception?

Reply:
agree, seems wired and useless 
@jialigit would you paste the sonar issue link here and let's know which issue sonar found about this change?

Reply:
yeah, I will add the issue link here and for others.

Reply:
> only private ctor is enough, why add the exception?

As suggestion by Sonar, this makes it more readable, and I accept it.

Reply:
I don't think that is useful, let's skip this one and only use the private constructor

Reply:
Ok, will follow it.